pkg/ui/ccl/src/views/reports/containers/stores/encryption.tsx

```diff
@@ -14,16 +28,77 @@ export default class EncryptionStatus extends React.Component<EncryptionStatusPr
     );
   }
 
-  render(): React.ReactElement<any> {
+  renderKey(isStoreKey: boolean, key: protos.cockroach.ccl.storageccl.engineccl.enginepbccl.KeyInfo$Properties) {
+    // Get the enum name from its value (eg: "AES128_CTR" for 1).
+    const encryptionType = protos.cockroach.ccl.storageccl.engineccl.enginepbccl.EncryptionType[key.encryption_type];
```

Comment:
@vilterp, @BramGruneir: FYI since we spent so much time looking for it: this works too to get the enum's string value. Good thing too because using `__proto__` makes `ui-lint` fail.

Reply:
Huh, not sure how we missed this. Glad you found it; `__proto__` kind of sketchy (also glad it made the linter fail).

Reply:
what were you guys trying to do with `__proto__`???